Tests/Common/IntegrationTestFixtureBase.Asserts.cs

```diff
@@ -103,7 +103,7 @@ public async Task AssertNetworkServerModuleLogStartsWithAsync(string logMessageS
                 new SearchLogOptions(string.Concat(logMessageStart1, " or ", logMessageStart2)));
         }
 
-        public async Task<SearchLogResult> SearchNetworkServerModuleAsync(Func<string, bool> predicate, SearchLogOptions options = null) =>
+        public async Task<SearchLogResult> SearchNetworkServerModuleAsync(Func<string, bool> predicate, SearchLogOptions options) =>
```

Comment:
I would argue that making the `SearchLogOptions` not `null` is not enough to guarantee that the `Description` is always set. To enforce setting the `Description` always we should disallow the no-args constructor and validate that the `description` is not null or empty when constructing the `SearchLogOptions`.

Reply:
Fair point, done this in fc631626d0e6c7a52dd6aea24183a4d727d99062